app/Services/Email.php

```diff
@@ -138,25 +138,36 @@ protected function setupEmail()
             $this->allMessages[$key]['user'] = $user;
 
             $tokens = $this->defineTokens($user);
-            $this->allMessages[$key]['message'] = $this->processMessage($tokens, $this->message);
 
-            if ($this->message->type == 'leaderboard') {
-                $this->allMessages[$key]['message']['leaderboard'] = $this->generateLeaderboard($this->users);
+            $message = [];
+
+            $message = $this->processMessage($tokens, $this->message);
 
-                // @TODO - get top 3 reportbacks
+            if ($this->message->type == 'leaderboard') {
```

Comment:
triple equals! Aiming to keep us all using triple equals as much as possible. Keeps us honest 😉